<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>consult</class>
 <widget class="QMainWindow" name="consult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consultar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>160</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #D52424;
    color: white;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #B71C1C;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="lbl_description">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>481</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Descripcion</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_ra">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>481</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Resultado esperado</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_consult">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>30</y>
         <width>91</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Consultar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_export">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>90</y>
         <width>91</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Exportar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_delete">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>30</y>
         <width>91</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_project">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>481</width>
         <height>25</height>
        </rect>
       </property>
       <property name="placeholderText" stdset="0">
        <string/>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_loop">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_id">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>101</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_tester">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>40</y>
         <width>231</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cb_case">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>481</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_cancel">
       <property name="geometry">
        <rect>
         <x>640</x>
         <y>90</y>
         <width>91</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="lbl_fotopaso">
            <property name="text">
             <string>Evidencia</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #D52424;
    color: white;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #B71C1C;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="btn_next">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>300</y>
            <width>31</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_back">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>300</y>
            <width>31</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
         <widget class="QTextEdit" name="txt_pasodesc">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>90</y>
            <width>191</width>
            <height>191</height>
           </rect>
          </property>
          <property name="placeholderText">
           <string>Descripcion del paso</string>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_update">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>300</y>
            <width>91</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>Actualizar</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
